head/src/main/kotlin/io/qalipsis/core/head/campaign/AbstractCampaignExecutor.kt

```diff
@@ -65,7 +66,7 @@ internal abstract class AbstractCampaignExecutor<C : CampaignExecutionContext>(
     private val campaignExecutionContext: C,
     private val campaignConstraintsProvider: CampaignConstraintsProvider,
     private val campaignHooks: Collection<CampaignHook>,
-    private val lockProvider: LockProvider,
+    private val lockProvider: LockProvider = InMemoryLockProviderImpl(),
```

Comment:
Still inject the LockProvider interface, otherwise there will be changes when a new implementation will be created